File: project-code/cs50x/problem-set-1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // Prompt the user for a valid height
    do
    {
        height = get_int("Enter the height of the pyramid: ");
    }
    while (height <= 0);

    // Print the pyramid
    for (int i = 1; i <= height; i++)
    {
        // Print spaces
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        // Print hashes
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: project-code/cs50x/problem-set-6/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    # Prompt user for the height of the pyramid
    height = 0
    while height < 1 or height > 8:
        height = get_int("Height: ")

    # Print the full pyramid
    for i in range(1, height + 1):
        # Calculate the number of spaces, left hashes, and right hashes
        spaces = height - i
        hashes = i
        print(" " * spaces + "#" * hashes + "  " + "#" * hashes)


if __name__ == "__main__":
    main()
